charts/rancher-wins-upgrader/scripts/upgrade.ps1

```diff
@@ -47,9 +47,15 @@ $winsUpgradeFilename = Split-Path -Path $winsUpgradePath -Leaf
 Create-Directory -Path $winsUpgradeDir
 Transfer-File -Src "c:\Windows\wins.exe" -Dst $winsUpgradePath
 
-Create-Directory -Path "c:\host\etc\rancher\wins"
-Transfer-File -Src $winsUpgradePath -Dst "c:\host\etc\rancher\wins\$winsUpgradeFilename"
-Transfer-File -Src "c:\scripts\config" -Dst "c:\host\etc\rancher\wins\config"
+if ($env:WINDOWS_HOST_PATH) {
```

Comment:
Comment for most of the PR: will this variable always end with a path separator (`\`)?

Reply:
Yep, I am setting it so it does. I create that in line 49 of daemonset.yaml and the function that is called is in the _helpers.tpl.

Reply:
Regardless of user input, it'll always end with the path separator?

Reply:
Hmm, good point. I pull what is placed in the config so there isn't user input. It may be potentially be left off.

Reply:
We control the windows hostpath and that will never have user input since that is inside our container.